plugins/eventMacro/eventMacro/Condition.pm

```diff
@@ -114,9 +114,4 @@ sub is_unique_condition {
 	0;
 }
 
-# Default: true
```

Comment:
Why?

This hurts the concept of an abstract class

Reply:
it's because this class will not exist anymore with this Pull Request

Reply:
it's because this class will not exist anymore with this Pull Request